mppm/release-notes/release-notes.md

```diff
@@ -4,6 +4,16 @@ title: Release notes
 description: Release notes for Multiplayer Play Mode including new features, updates, bug fixes, and known issues.
 ---
 
+## [1.4.0-pre.1] - 2024-12-05
+
+### Fixed
+- Virtual Player windows correctly update and stay in sync when you enter and exit Play mode.
+- Removed instances of outdated internal build profile API functions.
+
+### Added
+- Added a Scenario Launch window that displays the scenario stage, progress bars, and messages.
```

Comment:
How do you open this window?
Can you actually launch a scenario from this window?
When does this window appear?
progress bar for what?
Are the messages vivox?

Reply:
> How do you open this window? Can you actually launch a scenario from this window? When does this window appear? progress bar for what? Are the messages vivox?

Hi @Vic-Cooper , thanks for the questions, so, this window is a pop-up that displays the progress of scenario launching. It appears whenever playmode is entered from a scenario. The window features three progress bars, each representing one of the three stages of the scenario launching process: `Preparing`, `Deploying`, and `Launching`. Once all instances have completed launching (and entered playmode), the window closes automatically.

The messages are not from Vivox : P they are related to the scenario configuration. To be more specific, the messages indicate how many instances of each type (e.g., main editor, (virtual)editor instance, local instance, or remote instance) are currently launching within the scenario. Here is a screenshot of this pop-up, feel free to let me know if you have any further questions.

It might be more accurate to say: Added a Scenario Launching pop-up window that shows the scenario stages, progress bars, and configuration details.

<img width="397" alt="Screenshot 2024-12-16 at 09 23 06" src="https://github.com/user-attachments/assets/8dd54acf-bbc6-466e-b5d2-9b1680111e1a" />

Reply:
```suggestion
- Added a Scenario Launching pop-up window that shows the scenario stages, progress bars, and configuration details.```

Reply:
```suggestion
- Added a Scenario launching pop-up window that shows the scenario stages, progress bars, and configuration details.
```